_posts/2017-02-06-on-web-typography-review.md

```diff
@@ -0,0 +1,146 @@
+My favorite was word association, something we may all be familiar with, but he
+builds this process out in a way that seems easy and effective.
+
+Another important skill covered is pairing typefaces, something that poses a
+challenge to many of us. Santa Maria suggests a balance between two typefaces'
+distinction from each other and cohesion with each other, demonstrated with
+numerous examples, and explained with reference to the type terminology and
+classification methods reviewed in previous chapters.
+
+## What's next?
+
+After finishing the book, I had a couple of questions:
+
+### How can I actively improve my type design skills?
+
+After reading design resources, I usually feel armed with new information but
+unsure of how to actually apply it. I've found that improving design skills
+isn't something that happens instantly after you've learned something new
+— it takes practice!
+
+One of the methods I've found most helpful is to keep an eye out for
+design elements I like, and when I see them, break them down and figure out
+why they work. For typography, this would mean identifying the context,
+classifying the typeface, evaluating the letterforms to identify their physical
+characteristics and the feelings they invoke, and (this is key)
+bookmarking the source in an "Inspiration" folder. [Entire books have been
+written about this process](http://austinkleon.com/steal/), so don't be afraid
+to be inspired by others' work!
+
+After perusing my Inspiration folder, I follow these guidelines when designing
+type:
+
+- Keep it simple. It's conventional to use a maximum of two typefaces, and often
+using typefaces within the same family or a single typeface in multiple styles
+gives you all the variety you need.
+- Try things out in context. This is recommended by Santa Maria in the book, and
+giving typefaces and pairings a shot on your actual site (or a local version) is
+crucial. Most paid fonts are available via free trials, so no excuses!
+
+### How can our team apply this knowledge?
+
+Much of the team at Savas Labs focuses on back-end development, but we all have
+a desire to be holistic teammates with some proficiency at every stage of the
+process. We don't all have the time to invest in becoming typography experts
+— one can spend an entire career doing so — so we wanted to find ways to
+enable our team to be able to consult on typography and make informed decisions
+on type design when needed.
+
+To this end, we created a repository where we can continue to house a list of
+go-to typefaces, tried-and-tested pairings, favorite foundries, and excellent
+type resources. Curating this list and adding to it whenever we find something
+we like will provides with an easy reference so we're not starting from scratch
+every time, and is a nice way to practice the skills suggested above.
+
+## Closing thoughts
+
+I would highly recommend checking out *On Web Typography*, especially if you're
```

Comment:
I'd say link to the book one more time here, just to make it convenient for the readers to locate it quickly.